finetune/network_modules.py

```diff
@@ -312,3 +312,37 @@ def seq_lab_internal(hidden):
                 'transition_matrix': transition_params
             }
         }
+
+def cosine_similarity(hidden_0, hidden_1, targets, n_targets, dropout_placeholder, config, train=False, reuse=None, **kwargs):
```

Comment:
Can somebody explain this too me, my understanding is that you'd have something that would look like. 
```
cos_sim_logits = tf.reduce_sum(tf.multiply(hidden1, hidden2))
loss = tf.nn.sigmoid_cross_entropy_with_logits(labels=targets, logits=cos_sim_logits)
```
where targets shape is [batch, 1]